apix-docker/73/build.gradle

```diff
@@ -0,0 +1,8 @@
+dependencies {
+    baseAlfrescoWar platform("org.alfresco:acs-packaging:7.3.0.1")
+    baseAlfrescoWar 'org.alfresco:content-services@war'
+}
+
+dockerAlfresco {
+    baseImage = 'docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:7.3.0.1'
```

Comment:
can be updated to use `private.docker.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise`.
Also, I thought the tag with the label component was not available?

Reply:
Yes, I think I'll migrate to GHA, but seems cleaner in a separate ticket.